### ISSUE TITLE
Pagination is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6861]
The Pagination component is not accessible with Windows High Contrast Mode enabled in either Firefox or Edge.

## Environment

> Operating system

Windows 10

> Browser

Firefox, Edge

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

![image](https://user-images.githubusercontent.com/40970507/93650260-a6cba100-f9d3-11ea-90e1-1f05a2ea6c19.png)
